kythe/go/util/cmdutil/cmdutil_test.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Is this a test or an example? If an example, I think we should rename the file and add a file/package level doc explaining that.

Reply:
> Is this a test or an example?

It's both. The `testing` package also supports testable examples, which have the dual benefit of testing the code (or at least exercising it) and demonstrating its use. 

https://blog.golang.org/examples